test/tasks/ods-start_test.go

```diff
@@ -76,3 +79,13 @@ func TestTaskODSStart(t *testing.T) {
 
 	}
 }
+
+func checkFileContent(t *testing.T, wsDir, filename, want string) {
```

Comment:
@michaelsauter What do you think of moving this private func to `common_test.go`, so that any test file can reference it and avoid code duplication?

Reply:
@felipecruz91 yes, we will probably have a few helpers like this, and a common file for them would be good. I'll change this. note however that in Go the scope is per package, not per file. Files don't really matter :)